4주차/[Lv.3]_등굣길/주다현.js

```diff
@@ -0,0 +1,17 @@
+//런타임 에러가 나는 코드랍니다... 아직 해결하지 못했어요ㅠㅠㅠㅠ
+
+function solution(m, n, puddles) {
+  const arr = Array.from(Array(n), () => Array(m).fill(1));
```

Comment:
```suggestion
  const arr = Array.from(Array(n), () => Array(m).fill(0));
```
동적계획법이므로 [메모이제이션](https://ko.wikipedia.org/wiki/%EB%A9%94%EB%AA%A8%EC%9D%B4%EC%A0%9C%EC%9D%B4%EC%85%98)을 활용해야합니다. 따라서 우선 배열의 모든 값을 0으로 초기화합니다. 🤔